components/form-toggle/index.js

```diff
@@ -21,27 +22,35 @@ class FormToggle extends Component {
 	}
 
 	render() {
-		const { className, checked, children, onChange = noop } = this.props;
-		const id = 'toggle-' + this.id;
+		const {
+			className,
+			checked,
+			onChange = noop,
+			showHint = true,
+			id = 'toggle-' + this.id,
```

Comment:
Should we need to suffix this or manipulate it in any way, or can we treat it as verbatim presumed unique from the parent passing the prop?